src/ccl_tracers.c

```diff
@@ -348,14 +348,17 @@ void ccl_get_lensing_mag_kernel(ccl_cosmology *cosmo,
           chi = chi_arr[ichi];
           a = ccl_scale_factor_of_chi(cosmo, chi, &local_status);
           z = 1./a-1;
+          // mgfac = 1.0;
```

Comment:
some of these commented-out lines probably need deleting?

Reply:
removed.